render-configs.sh

```diff
@@ -102,11 +102,13 @@ GOOGLE_SA_CERT=/tmp/jade-dev-account.pem
 if [[ "${RBS_ENV}" == "tools" ]]; then
     BUFFER_CLIENT_SERVICE_ACCOUNT_VAULT_PATH=secret/dsde/terra/kernel/integration/tools/buffer/client-sa
     BUFFER_CLIENT_SERVICE_ACCOUNT_GSM_PROJECT=broad-dsde-qa
+    RBS_GSM_SECRET=buffer-client-sa-b64-integration
```

Comment:
One thing about the name; note that other env vars like `RBS_POOLID` are called that because they correspond to properties in `ResourceBufferServiceConfiguration` (and are also mapped in `application.properties` as `rbs.poolId`). So maybe this shouldn't use `RBS_` since it doesn't correspond to a configuration property name.

Reply:
Good shout -- updated!